index.js

```diff
@@ -0,0 +1,77 @@
+function validatePassword (pwd) {
+    //* Must be a minimum of 8 characters (8 characters or more) 
+    if (pwd.length < 8) {
+        return false
+    }
+    //* Must contain at least one lower case letter
+    const lowerCase = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+    let hasLowerCase = false;
+    for (let i = 0; i < pwd.length; i++) {
+        const char = pwd.charAt(i);
```

Comment:
Notice how these two lines are repeated four total times in your code file. That is typically an indicator that the repeating code can be refactored to run just one time, often in its own function that can then be fed any data/variables that it needs to interact with or alter the value(s) of.

For example, in the solution branch the index.js file contains the following code:
`
function isLowerCase(pwd, index) {
  return pwd.charCodeAt(index) >= 97 && pwd.charCodeAt(index) <= 122
}

function isUpperCase(pwd, index) {
  return pwd.charCodeAt(index) >= 65 && pwd.charCodeAt(index) <= 90
}

function isNumeric(pwd, index) {
  return !isNaN(pwd[index])
}

function validatePassword(pwd) {
  if (pwd.length < 8) return false

  let upper = 0
  let lower = 0
  let numeric = 0
  let special = 0

  for (let i = 0; i < pwd.length; i++) {
    if (isLowerCase(pwd, i)) {
      lower++
    } else if (isUpperCase(pwd, i)) {
      upper++
    } else if (isNumeric(pwd, i)) {
      numeric++
    } else {
      special++
    }
  }

  return lower > 0 && upper > 0 && numeric > 0 && special > 0
}

module.exports = validatePassword
`